tests/bookmarks_and_history/test_toggle_bookmarks_toolbar.py

```diff
@@ -0,0 +1,41 @@
+from time import sleep
+
+import pytest
+from selenium.webdriver import Firefox
+
+from modules.browser_object_navigation import Navigation
+
+
+@pytest.fixture()
+def test_case():
+    return "2084637"
+
+
+def test_toggle_bookmark_toolbar(driver: Firefox):
+    """
+    C2084637: Verify that the user can Hide or Show the Bookmarks Toolbar using Keyboard shortcuts
+    """
+    # instantiate object
+    nav = Navigation(driver).open()
+
+    def is_bookmarks_toolbar_collapsed() -> str:
```

Comment:
We could do this with:
```python
nav.element_attribute_contains("bookmarks-toolbar", "collapsed", "true")
```

This would eliminate the need for the hard waits as it uses `wait.until()` under the hood.